### PR TITLE
quick shell to transfer & createAccount & deployContract & callContract

### LINKED ISSUE #233: Quick shell to play xuperchain
**Brief of your idea**
Use shell command to have a quick start with xuperchain, you just need to transmit necessary parameters.

**What could be better if XuperChain using your idea**
It is more easier to have a start with xuperchain.

### PR DESCRIPTION
**Pull request by ToWorld**
## Description

What is the purpose of the change?
quick start to play xuperchain

Fixes #233

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Brief of your solution
Wrap all necessary command in the shell file, client only needs to give paramters.

## How Has This Been Tested?

None

### CODE REVIEW

File: cmd/quick_shell/testnet.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ip_port=""
+public_key=""
```
Comment (yucaowang) [inline cmd/quick_shell/testnet.conf]: is public_key referring to compliance service's address?

File: cmd/quick_shell/deployContract.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+if [ ! -d "$acl" ]; then
+    mkdir $acl
+fi
+addrs="./data/acl/addrs"
+if [ ! -f "$addrs" ]; then
+    touch $addrs
+fi
+echo $public_key > data/acl/addrs
+echo "$accountName/$address" >> data/acl/addrs
+./xchain-cli wasm deploy --account $accountName --cname $contractName -H $ip_port -m $contractNamePath --arg '{"creator":"xchain"}' --output contractRawTx.out --fee $fee
```
Comment (yucaowang) [inline cmd/quick_shell/deployContract.sh]: "arg" should be a custom field

File: cmd/quick_shell/testnet.conf (hunk 1 above)
Comment (ToWorld) [reply cmd/quick_shell/testnet.conf]: Yes.

File: cmd/quick_shell/deployContract.sh (hunk 2 above)
Comment (ToWorld) [reply cmd/quick_shell/deployContract.sh]: You are correct.

**Pull request merged**